stylesheets/style.css

```diff
@@ -1,7 +1,139 @@
-/*
-background-grey: #F4F3F4;
-button-purple: #192592
-main-titles dark-grey: #2C303F;
-paragraph grey: #5b5e6d;
-nav grey: #5b5e6d;
-*/
+
+*{
+  margin:0;
+  padding:0;
+}
+
+
+@media (max-width: 768px) {
```

Comment:
You don't need to put all your code inside this @mediaquery. Just specific things that only are different for that screen size. Please if you have questions regarding how to use media queries let us know.